docs/technology_types_guide/CITE_seq/Overview.md

```diff
@@ -0,0 +1,59 @@
+# CITE-seq: Cellular Indexing of Transcriptomes and Epitopes by Sequencing
+
+### Method
+
+CITE-seq enables researchers to simultaneously capture the transcriptome and surface protein expression on the same cells with next generation sequencing technology. This can be done using DNA-barcoded antibodies which convert the detection of 1 or more surface proteins into a quantitative, sequenceable readout.
+
+Specifically, a set of DNA-barcoded antibodies are bound to specific cell surface protein(s). The cell and bound antibody are then encapsulated in a droplet alongside a poly-dT oligo-coated bead, and the cell is lysed. Reverse transcription and cDNA amplification of the antibody-bound DNA barcode and cellular mRNA transcripts then occurs, and transcripts are indexed with a cell barcode. This is illustrated below in **Figure 1**.
+
+Once scRNA-seq libraries are sequenced, transcripts can be mapped back to individual cells using the cell barcode. The presence of a particular single cell surface protein can also be identified by the antibody barcode. The scRNA-seq library preparation method used for CITE-seq is typically 10X Genomics, Drop-seq or other oligodT-based scRNA-seq methods.
+
+
+<center><u>Figure 1</u>: Illustration of CITE-seq</center>
+<br
+    </br>
+
+<img src="https://github.com/ebi-ait/hca-ebi-wrangler-central/blob/Add-technology-type-info-folder/technology_types_explanors/CITE_seq/visuals/CITE-seq.png" width="700" height="500">
```

Comment:
1. This link is broken.
2. Please use relative links
3. Verify all other image links.

Reply:
@amnonkhen relative links behave poorly in github.io (https://ebi-ait.github.io/hca-ebi-wrangler-central/ ,where this SOP will be shown)